ui/src/dashboards/components/Dashboard.js

```diff
@@ -17,11 +20,14 @@ const Dashboard = ({
   inPresentationMode,
   onOpenTemplateManager,
   onSummonOverlayTechnologies,
+  onSelectTV,
```

Comment:
nit: can we make this `onSelectTempVar` or something more descriptive?  I understand not wanting to type TemplateVariable but TV makes me think of television